index.html

```diff
@@ -0,0 +1,126 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <!--
+
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+    
+
+    -->
+    <div class="container">
+      <div>
+        <h1 id="title" class="center">freeCodeCamp Survey Form</h1>
+        <p id = "description" class="center">Thank you for taking the time to help us improve the platform</p>
+      </div>
+      <form id="survey-form">
+      
+            <div class="form-group">
+        <label id="name-label">Name</label>
+        <input type="text"
+        name = "name"
+        class="form-item"
+        id = "name"
+        placeholder="Enter your Name" >
+      </div>
```

Comment:
I guess it would be better to use the semantic tag.
And I think it would be better if you use to make it required and other important fields like this.

Reply:
- [ ] Awesome inline comment!!! But can you please clarify further which of the semantic tag will be more appropriate. Thanks 👍